### PR TITLE
MariaDB and MySQL support

### LINKED ISSUE #16: Database handler: MySQL
This should provide support to make MySQL database backups.

### LINKED ISSUE #56: Database: MariaDB support
We'll need to add support for MariaDB, as this database is now going to be part of our stack at PythonDiscord.

### PR DESCRIPTION
**Pull request by matacoder**
As MariaDB and MySQL have common roots we can use the same algorithm to backup them using `mysqldump`.
I've integrated it in Dockerfile of blackbox as part of MariaDB package
I added on docker-compose two new containers for that databases and introduced a `heathcheck` for every database so blackbox will start backup only when all databases ready.

Resolves #56
Resolves #16

### CODE REVIEW

**Comment by lemonsaurus** [comment]
@matacoder 

A MariaDB pull request is not the right place to change the versioning scheme we use.

I don't see any compelling reason to change from SemVer to CalVer, so I'm opposed to that change. Please revert that change from this branch.

Remember to try to keep pull requests as small as possible. It's better to make two pull requests rather than two different changes in one pull request.

That said, I'll review this in the near future.

**Comment by matacoder** [comment]
Okay. As small as possible.

File: docker-compose.yaml [hunk 1]
```diff
@@ -28,9 +23,61 @@ services:
...
+      postgres:
+        condition: service_healthy
+      mongo:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      maria:
+        condition: service_healthy
+      mysql:
+        condition: service_healthy
```
Comment (lemonsaurus) [inline docker-compose.yaml:83]: I did not know you could do this, but I really like it. 👏🏼

**Review by lemonsaurus — APPROVED** [review]
Looks perfect!

**Pull request merged**